apps/configurator/lib/configurator_web/controllers/map_configuration_html/map_configuration_form.html.heex

```diff
@@ -9,6 +9,7 @@
     label="Version"
   /> <.input field={f[:name]} type="text" label="Name" />
   <.input field={f[:radius]} type="number" label="Radius" step="any" />
+  <.input field={f[:active]} type="select" options={[true, false]} label="Active" />
```

Comment:
This could be changed for a checkbox and we can move it at the top of the form

Reply:
Sounds good!